sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -46,7 +46,7 @@ public BlobTestBase(bool async, BlobClientOptions.ServiceVersion serviceVersion,
 
         public BlobClientOptions GetOptions(bool parallelRange = false)
         {
-            var options = new BlobClientOptions(_serviceVersion)
+            var options = new BlobClientOptions()
```

Comment:
We shouldn't remove this but instead add V2019_07_07 to the ClientTestFixture attribute up top so we test both version of the API.  See https://github.com/Azure/azure-sdk-for-net/pull/9582 for more details.

Reply:
fixed.